proto/src/determined/api/v1/master.proto

```diff
@@ -97,6 +97,10 @@ message ResourceAllocationAggregatedRequest {
 }
 // Response to ResourceAllocationAggregatedRequest.
 message ResourceAllocationAggregatedResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "resourceEntries" ] }
```

Comment:
@dzhu can you please check if this change and the one in `proto/src/determined/master/v1/master.proto` are fine?

Reply:
Looks good!